misc/dbt-materialize/CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
 # dbt-materialize Changelog
+## 1.1.4 - 2022-08-24
+
+* Enable cluster use for materialize cloud deployments.
+
+  ```sql
+  {{ config(materialize_cluster='not_default', materialized='materializedview') }}
+    SELECT ...
+  ```
+* Run tests against materialize/materialized:v0.26.4 and cloud(main).
```

Comment:
I'd strike this! The policy that we follow for release notes in the main repository is to only talk about things that have actual impact on end users.

```suggestion
```